### ISSUE TITLE
ClassCastException in BlobstoreUploadAction with last snapshot

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#160]
```
What steps will reproduce the problem?
1. Attempt to upload file with BlobstoreUploadAction in Dev Mode

What is the expected output? What do you see instead?
Upload fails with following exception:
java.lang.ClassCastException: java.util.ArrayList cannot be cast to 
java.util.Vector
    at gwtupload.server.gae.BlobstoreUploadAction.doPost(BlobstoreUploadAction.java:199)

What version of the product are you using? On what operating system?
gwtupload-gae-0.6.5-20120525.113743-9.jar
gwtupload-0.6.5-20120525.113719-12.jar
Firefox 14, Ubuntu
```

Original issue reported on code.google.com by `jakub.st...@czechgeeks.cz` on 23 Jul 2012 at 1:00

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 8c5691c007ef.
```

Original comment by `manuel.carrasco.m` on 30 Nov 2012 at 9:02
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 30 Nov 2012 at 9:14
- Added labels: **Milestone-0.6.5**
